audbackend/core/backend.py

```diff
@@ -530,12 +491,13 @@ def split(
         """
```

Comment:
`split()` is also missing a check for invalid chars in `path`

Reply:
Here I was not sure if we really need the check since we are actually not accessing the backend. But if you think we should, we can add such a check of course.

Reply:
You are right that it is not needed, but I think I would prefer to add it as we then have the same check whenever `path` is given. Such consistency is always easier to grasp for a user.

Reply:
Ok, we now check for invalid characters also in `Backend.split()` and `Backend.join()`.